sdcm/nemesis.py

```diff
@@ -1877,6 +1882,44 @@ def disrupt_grow_shrink_cluster(self):
 
         self.log.info("Finish cluster shrink. Current number of nodes %s", len(self.cluster.nodes))
 
+    def disrupt_hot_reloading_internode_certificate(self):
+        '''
+        https://github.com/scylladb/scylla/issues/6067
+        Scylla has the ability to hot reload SSL certificates.
+        This test will create and reload new certificates for the inter node communication.
+        '''
+        self._set_current_disruption('HotReloadInternodeCertificate')
+        if not self.cluster.params.get('server_encrypt', None):
+            raise UnsupportedNemesis('Server Encryption is not enabled, hence skipping')
+
+        @retrying(allowed_exceptions=LogContentNotFound)
+        def check_ssl_reload_log(target_node, file_path):
+            msg = target_node.remoter.run(f'journalctl -u scylla-server --since="{time_now}" | '
```

Comment:
`time_now` isn't defined in this scope, 
please add it to the parameters of `check_ssl_reload_log`

Reply:
It's enough for now, but might cause and issue once someone might refactor this function into a different place,
also how it's different from `target_node` and `file_path`, why for them you are not counting on the other context ?